src/MultiRegion/multi_region_split_explicit_free_surface.jl

```diff
@@ -40,11 +40,12 @@ end
 @inline augmented_kernel_offsets(grid, ::YPartition) = (0, - halo_size(grid)[2] + 1)
 @inline augmented_kernel_offsets(grid, ::CubedSpherePartition) = (- halo_size(grid)[2] + 1, - halo_size(grid)[2] + 1)
 
-function FreeSurface(free_surface::SplitExplicitFreeSurface, velocities, grid::MultiRegionGrids)
+# Internal function for HydrostaticFreeSurfaceModel
+function materialize_free_surface(free_surface::SplitExplicitFreeSurface, velocities, grid::MultiRegionGrids)
```

Comment:
This method is broken (already broken on `main` that is) and not used in tests. Should we delete it? Seems like we don't support `SplitExplicitFreeSurface` on `MultiRegionGrids` quite yet... Perhaps we replace it with an informative error?